### PR TITLE
dev:Wallet service testing is done.

### PR DESCRIPTION
**Pull request by kai890707**
**Description**

1. Wallet service testing is done.
2. Change the WalletController show method route, because the show method of the resource route needs to pass parameters but the user key is transmitted as a header.
3. Change the WalletController show method annotation.

**Checklist:**
- [x] Securely signed commits
- [x] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [x] Conforms to style guide

### CODE REVIEW

File: develop/app/Controllers/v2/WalletController.php [hunk 1]
```diff
@@ -54,7 +53,7 @@ public function show()
     public function create()
     {
         $data = $this->request->getJSON(true);
-
+
```
Comment (ping-yee) [inline develop/app/Controllers/v2/WalletController.php:56]: What does this new line do?

**Review by ping-yee — COMMENTED** [review]
Have a little change need to be handled.

File: develop/app/Controllers/v2/WalletController.php (hunk 1 above)
Comment (kai890707) [reply develop/app/Controllers/v2/WalletController.php]: In the original show method, there was a comment to mark the arguments to be passed in the show method, but after the show method was modified, there was no need to pass the arguments, so the comment was removed from this line,So the code goes up one line.

**Review by ping-yee — APPROVED** [review]
LGTM

**Pull request merged**